shellwords.gemspec

```diff
@@ -21,8 +21,9 @@ Gem::Specification.new do |spec|
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = spec.homepage
 
-  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  gemspec = File.split(__FILE__)
```

Comment:
Minor: Could assign the result of File.split to two local variables `gemspec_directory, gemspec_file`. Could make the expression below even easier to follow.

Reply:
Used `srcdir` since it is the source directory.